nixos/modules/flyingcircus/tests/nodejs7.nix

```diff
@@ -0,0 +1,30 @@
+import ../../../tests/make-test.nix ({ ... }:
+{
+  name = "nodejs7";
+
+  nodes = {
+    master =
+      { pkgs, config, ... }:
+      {
+
+        imports = [
+          ./setup.nix
+          ../static/default.nix
+          ../roles/default.nix
+          ../services/default.nix
+          ../platform/default.nix
+        ];
+
+        virtualisation.memorySize = 1024;
+        environment.systemPackages = [
+          pkgs.nodejs7
+        ];
+      };
+  };
+
+  testScript = ''
+    startAll;
+
+    $master->succeed('node -e \'console.log("ok")\' ');
```

Comment:
The test does not verify that this is actually nodejs7. Add another line which checks the output of `node --version`. On my test system, `node` points to nodejs 0.12.7.